### PR TITLE
feat: revamp the order history page

### PR DESCRIPTION
**Pull request by lukmannm**
Dekstop:
VA:
image
eWallet:
image
Manual Bank Transfer:
image
Qr Code:
image
OTC:
image

Mobile:
VA:
image

eWallet:
image

Manual Bank Transfer:
image

Qr Code:
image

OTC:
image

Toash:
image
image

### CODE REVIEW

File: locales/en/paymentMethod.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      "counterInstruction": "Show this barcode to",
+      "downloadQrSuccess": "Image Downloaded Successfully",
+      "expiryDateWarning": "Complete payment before",
+      "manualBankTransfer": "Manual Bank Transfer",
+      "manualBankTransferInstruction": "Choose 1 payment recipient account",
+      "page": "Payment page",
+      "qrDownload": "Download QR Code",
+      "qrInstruction": "Scan QR Code using e-wallet or m-banking app that supports QRIS payment",
+      "seeMore": "For more info, see",
+      "continuePayment": "Continue to Payment"
```
Comment (virginiaritonga) [inline locales/en/paymentMethod.ts]: sort ke atas yaa

File: locales/id/paymentMethod.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      "downloadQrSuccess": "Berhasil Mengunduh Gambar",
+      "expiryDateWarning": "Bayar sebelum",
+      "manualBankTransfer": "Transfer Bank Manual",
+      "manualBankTransferInstruction": "Pilih 1 rekening penerima pembayaran",
+      "manualBankTransferAccountNamePrefix": "a.n.",
+      "page": "halaman pembayaran",
+      "qrDownload": "Unduh QR Code",
+      "qrInstruction": "Scan kode QR melalui aplikasi e-wallet atau m-banking yang mendukung pembayaran QRIS",
+      "seeMore": "Lihat info lengkap di",
+      "continuePayment": "Lanjutkan pembayaran"
```
Comment (virginiaritonga) [inline locales/id/paymentMethod.ts]: sort ke atas yaa

**Review by virginiaritonga — APPROVED** [review]

File: pages/[lng]/account/index.tsx [hunk 3]
```diff
@@ -7,10 +7,10 @@ import {
   useI18n
 } from '@sirclo/nexus'
 import { toast } from 'react-toastify'
-import { CheckCircle } from 'react-feather'
+import { CheckCircle, Copy, Download } from 'react-feather'
```
Comment (mayun19) [inline pages/[lng]/account/index.tsx]:
```
{ 
   CheckCircle, 
   Copy, 
   Download 
}
```

klo 3 variable jdiin gini

File: public/scss/pages/Account.module.scss [hunk 4]
```diff
@@ -1920,4 +1962,229 @@
...
+
+    &_copyCodeButton 
+    {
+      background: none;
+      border: none;
+
+      svg 
+      {
+        width: 12px;
+        height: 12px;
```
Comment (mayun19) [inline public/scss/pages/Account.module.scss]: bisa pakai mixin

File: public/scss/pages/Account.module.scss [hunk 5]
```diff
@@ -1920,4 +1962,229 @@
...
+        background: $color_lighter_low_blue;
+        padding: 8px 8px;
+        border-bottom-left-radius: 4px;
+        border-bottom-right-radius: 4px;
+
+        svg 
+        {
+          color: $color_teal_blue;
+          width: 12px;
+          height: 12px;
```
Comment (mayun19) [inline public/scss/pages/Account.module.scss]: idem

File: public/scss/pages/Account.module.scss [hunk 6]
```diff
@@ -1920,4 +1962,229 @@
...
+        @include fontSize(16);
+        width: 180px;
+        margin-bottom: 10px;
+        border: none;
+        background-color: transparent;
+        color: $color_tiktok;
+
+        svg 
+        {
+          width: 14px;
```
Comment (mayun19) [inline public/scss/pages/Account.module.scss]: sama

File: public/scss/pages/Account.module.scss [hunk 7]
```diff
@@ -1920,4 +1962,229 @@
...
+      color: $color_gray_darker;
+      background: rgba(255, 152, 0, 0.1);
+      padding: 4px 6px;
+      border-radius: 4px;
+      
+      svg 
+      {
+        color: $color_orange;
+        width: 15px;
+        height: 15px;
```
Comment (mayun19) [inline public/scss/pages/Account.module.scss]: idem

File: public/scss/pages/Account.module.scss [hunk 8]
```diff
@@ -1920,4 +1962,229 @@
...
+      &_date 
+      {
+        font-weight: 400 !important;
+        font-size: 12px !important;
+        padding-left: 3px !important;
+      }
+
+      &_warning
+      {
+        font-weight: 400 !important;
```
Comment (mayun19) [inline public/scss/pages/Account.module.scss:2181]: bisa pake mixin

File: public/scss/pages/Account.module.scss [hunk 9]
```diff
@@ -1920,4 +1962,229 @@
...
+      {
+        color: $color_orange;
+        width: 15px;
+        height: 15px;
+        margin-right: 6px;
+      }
+
+      &_date 
+      {
+        font-weight: 400 !important;
```
Comment (mayun19) [inline public/scss/pages/Account.module.scss:2174]: bisa pake mixin

**Review by mayun19 — CHANGES_REQUESTED** [review]

**Review by mayun19 — APPROVED** [review]

File: public/scss/pages/Account.module.scss (hunk 8 above)
Comment (lukmannm) [reply public/scss/pages/Account.module.scss:2181]: izin resolve, seperti yang kita didiskusikan dichat

File: public/scss/pages/Account.module.scss (hunk 9 above)
Comment (lukmannm) [reply public/scss/pages/Account.module.scss:2174]: idem

**Review by virginiaritonga — APPROVED** [review]

**Review by feliciakri — APPROVED** [review]

**Pull request merged**